### PR TITLE
Add titles to iframes

### PR DESCRIPTION
**Pull request by stephaniehobson**
If there is a preferred way to make these translatable please point me to an example and I will integrate it :)

### CODE REVIEW

File: macros/EmbedGHLiveSample.ejs [hunk 1]
```diff
@@ -8,10 +8,11 @@
 //
 //  Example call {{EmbedGHLiveSample("canvas-raycaster", 900, 300)}}
 //
- 
+
 var str = "";
 var widthHeight = "";
 var url = "https://mdn.github.io/" + $0;
+var title = 'Live Sample.';
```
Comment (ExE-Boss) [inline macros/EmbedGHLiveSample.ejs]: This should be:
```js
let s_title = MDN.localString({
  "en-US": "MDN Web Docs Live Sample",
});
```

File: macros/EmbedInteractiveExample.ejs [hunk 2]
```diff
@@ -11,6 +11,7 @@
 let iClass = "interactive";
 let url = kuma.url.resolve(env.interactive_examples.base_url, $0);
 let siteURL = mdn.siteURL();
+let title = 'Interactive example.'
```
Comment (ExE-Boss) [inline macros/EmbedInteractiveExample.ejs]: This should be:
```js
let s_title = MDN.localString({
  "en-US": "MDN Web Docs Interactive Example",
});
```

File: macros/EmbedInteractiveExample.ejs [hunk 3]
```diff
@@ -28,4 +29,4 @@ if ($1) {
 }
 
 %>
-<iframe class="<%= iClass %>" width="100%" height="250" frameborder="0" src="<%= url %>" title="MDN Web Docs Interactive Example"></iframe>
+<iframe class="<%= iClass %>" width="100%" height="250" frameborder="0" src="<%= url %>" title="Interactive Example"></iframe>
```
Comment (ExE-Boss) [inline macros/EmbedInteractiveExample.ejs]: `title="MDN Web Docs Interactive Example"` should be changed to `title="<%= s_title %>"`

File: macros/EmbedLiveSample.ejs [hunk 4]
```diff
@@ -21,6 +21,7 @@ var screenshotUrl = $3;
 var sampleSlug = $4;
 var className = $5 || "sample-code-frame";
 var allowedFeatures = $6;
+var title = 'Live Sample.';
```
Comment (ExE-Boss) [inline macros/EmbedLiveSample.ejs]: This should be:
```js
let s_title = MDN.localString({
  "en-US": "MDN Web Docs Live Sample",
});
```

File: macros/EmbedNewRelicChart.ejs [hunk 5]
```diff
@@ -7,6 +7,6 @@
 
 var height = $1 || 300;
 
-var s_result = '<p><iframe allowfullscreen="allowfullscreen" scrolling="no" frameborder="0" height="' + height + '" src="https://rpm.newrelic.com/public/charts/' + $0 + '" width="100%"></iframe></p>';
+var s_result = '<p><iframe title="New relic chart." allowfullscreen="allowfullscreen" scrolling="no" frameborder="0" height="' + height + '" src="https://rpm.newrelic.com/public/charts/' + $0 + '" width="100%"></iframe></p>';
```
Comment (ExE-Boss) [inline macros/EmbedNewRelicChart.ejs]: This should be:
```js
let s_title = MDN.localString({
  "en-US": "MDN Web Docs Live Sample",
});

let s_result = '<p><iframe title="' + s_title + '" allowfullscreen="allowfullscreen" scrolling="no" frameborder="0" height="' + height + '" src="https://rpm.newrelic.com/public/charts/' + $0 + '" width="100%"></iframe></p>';
```

File: macros/EmbedSVG.ejs [hunk 6]
```diff
@@ -2,16 +2,16 @@
 var found,
     width  = $1 > 0 ? $1 : 250,
     height = $2 > 0 ? $2 : 250;
- 
+
```
Comment (ExE-Boss) [inline macros/EmbedSVG.ejs]: Add the following before this line:
```js
let s_title = MDN.localString({
  "en-US": "MDN Web Docs Embedded SVG",
});
```

File: macros/EmbedSVG.ejs [hunk 7]
```diff
@@ -2,16 +2,16 @@
...
 env.files.forEach(function (aFile) {
     if(!found && $0 === aFile.filename) {
         found = true;
 %>
-<iframe src="<%- aFile.url %>" width="<%- width %>" height="<%- height %>" frameborder="0" sandbox="allow-scripts"></iframe>
+<iframe title="Embeded SVG." src="<%- aFile.url %>" width="<%- width %>" height="<%- height %>" frameborder="0" sandbox="allow-scripts"></iframe>
```
Comment (ExE-Boss) [inline macros/EmbedSVG.ejs]: `title="Embedded SVG."` should be changed to `title="<%= s_title %>"`

File: macros/EmbedYouTube.ejs [hunk 8]
```diff
@@ -25,7 +25,7 @@ if (ratio == "16:9") {
 
 var video = "<div class='intrinsic-wrapper'>";
 video += "<div class='intrinsic-container " + ratio_class + "'>";
-video += "<iframe allowfullscreen src='https://www.youtube.com/embed/" + $0 + "?rel=0&html5=1'></iframe>";
+video += "<iframe title='Video.' allowfullscreen src='https://www.youtube.com/embed/" + $0 + "?rel=0&html5=1'></iframe>";
```
Comment (ExE-Boss) [inline macros/EmbedYouTube.ejs]: Change `title='Video.'` to `title="<%= s_title %>"` and add the following after line 15:
```js
let s_title = MDN.localString({
  "en-US": "YouTube Video",
});
```

**Comment by stephaniehobson** [comment]
Adding titles to the iframes was one of the things @estelle identified in her accessibility audit for the HackOnMDN weekend. [Titles make iframes more accessible](https://webaim.org/techniques/frames/#title). What are your concerns about adding them?

People will already know that they are on the MDN site so I don't recommend repeating that information in the titles. Unless you have a specific reason in mind for including it?

A period at the end of the title will make screen readers pause before continuing to read so I've included one in the strings.

Thanks for the examples on how to localize. I've updated the PR with the localizeable strings. I matched the coding style of the files each string is in (I used var if the file used var for other variables and let if the file used let). I can change that if it's a problem though.

File: macros/EmbedSVG.ejs [hunk 9]
```diff
@@ -1,17 +1,20 @@
 <%
 var found,
     width  = $1 > 0 ? $1 : 250,
-    height = $2 > 0 ? $2 : 250;
- 
+    height = $2 > 0 ? $2 : 250,
+    s_title = mdn.localString({
+        "en-US": 'Embeded SVG..'
```
Comment (ExE-Boss) [inline macros/EmbedSVG.ejs]: There are two `.`s at the end of the string when there should be only one.

**Pull request closed without merging**